test/components/portal.spec.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import {selectDom} from 'test-drive';
+import { ClientRenderer, expect, waitFor } from 'test-drive-react';
+import {Portal} from '../../src';
+import {PortalFixture} from '../fixtures/portal-fixture';
+import {sleep} from '../utils/sleep';
+
+const portal = 'PORTAL';
+
+describe('<Portal />', function() {
+    const clientRenderer = new ClientRenderer();
+    const bodySelect = selectDom(document.body);
+
+    afterEach(function() {clientRenderer.cleanup(); });
+
+    it('displays the portal and renders its children if the open prop is given', async function() {
+        clientRenderer.render(
+            <Portal
+                open={true}
+            >
+                <span data-automation-id="SPAN">Portal Body</span>
+            </Portal>);
+
+        await waitFor(() => {
+            expect(bodySelect(portal)).to.be.present();
+            expect(bodySelect(portal, 'SPAN')).to.be.present();
+        });
+    });
+
+    it('does not add the portal to the DOM if open is false', async function() {
+        clientRenderer.render(
+            <Portal
+                open={false}
+            >
+                <span data-automation-id="SPAN">Portal Body</span>
+            </Portal>);
+        await sleep(500);
+        await waitFor(() => {
+            expect(bodySelect(portal)).to.not.exist;
+            expect(bodySelect(portal, 'SPAN')).to.not.exist;
+        });
+    });
+
+    it('removes the portal if the props update', async function() {
```

Comment:
description is not good. maybe "should remove portal from DOM when open prop is changed to false"